action.yml

```diff
@@ -12,6 +12,10 @@ inputs:
     description: 'Enter an existing Slot other than the Production slot'
     required: false
     default: 'production'
+  commit-message:
+    description: 'commit message'

```

Comment:
can you add more explicit description like where it will be used and user don't required to set this input